File: example-projects/simple-setup/maven/src/test/java/com/example/FuzzTestCase.java
package com.example;

import com.code_intelligence.jazzer.junit.FuzzTest;
import com.code_intelligence.jazzer.mutation.annotation.NotNull;

public class FuzzTestCase {
    @FuzzTest
    void myFuzzTest(int a, int b, @NotNull String c) {
        ExploreMe ex = new ExploreMe(a);
        ex.exploreMe(b, c);
    }
}

File: example-projects/simple-setup/gradle-kotlin/src/test/kotlin/com/example/FuzzTestCase.kt
package com.example

import com.code_intelligence.jazzer.junit.FuzzTest
import com.code_intelligence.jazzer.mutation.annotation.NotNull

class FuzzTestCase {
    @FuzzTest
    fun myFuzzTest(a: Int, b: Int, @NotNull c: String) {
        val ex = ExploreMe(a)
        ex.exploreMe(b, c)
    }
}
